src/app/modules/pages/components/sign-up-form/sign-up-form/sign-up-form.component.ts

```diff
@@ -26,6 +18,12 @@ export class SignUpFormComponent {
 		confirmPassword: new FormControl('', Validators.required),
 	});
 
+	private readonly notifier: NotifierService;
+
+	constructor(private httpClient: HttpClient, notifierService: NotifierService) {
```

Comment:
remove
```
public notifier: NotifierService
this.notifier = notifierService;
```

add in `constructor(private httpClient: HttpClient, private notifierService: NotifierService)`
and use this.notifierService